go-controller/pkg/ovn/base_network_controller_secondary.go

```diff
@@ -877,3 +885,38 @@ func (bsnc *BaseSecondaryNetworkController) requireDHCP(pod *corev1.Pod) bool {
 		bsnc.IsPrimaryNetwork() &&
 		bsnc.TopologyType() == types.Layer2Topology
 }
+
+func (bsnc *BaseSecondaryNetworkController) updateLogicalSwitchPortsForLiveMigrationPods(pod *corev1.Pod, lsp *nbdb.LogicalSwitchPort, ops []ovsdb.Operation, nadName, switchName string) ([]ovsdb.Operation, error) {
+	kubevirtLiveMigrationStatus, err := kubevirt.DiscoverLiveMigrationStatus(bsnc.watchFactory, pod)
+	if err != nil {
+		return nil, fmt.Errorf("failed to discover Live-migration status: %w", err)
+	}
+	if kubevirtLiveMigrationStatus != nil {
+		if pod.Name == kubevirtLiveMigrationStatus.TargetPod.Name {
+			lsps := []*nbdb.LogicalSwitchPort{lsp}
+			kubevirtLiveMigrationInReadyState := kubevirtLiveMigrationStatus.State == kubevirt.LiveMigrationTargetDomainReady
+			lsp.Enabled = ptr.To(kubevirtLiveMigrationInReadyState)
+			if kubevirtLiveMigrationInReadyState {
+				// closing the sourcePod lsp to ensure traffic goes to the now ready targetPod.
+				sourcePodLspName := bsnc.GetLogicalPortName(kubevirtLiveMigrationStatus.SourcePod, nadName)
+				sourcePodLsp := &nbdb.LogicalSwitchPort{Name: sourcePodLspName}
+				sourcePodLsp, err = libovsdbops.GetLogicalSwitchPort(bsnc.nbClient, sourcePodLsp)
+				if err != nil {
+					return nil, fmt.Errorf("cannot fetch switch port for node %s: %v", sourcePodLspName, err)
+				}
+				sourcePodLsp.Enabled = ptr.To(false)
+				lsps = append(lsps, sourcePodLsp)
+			}
+
+			logicalSwitch, err := libovsdbops.GetLogicalSwitch(bsnc.nbClient, &nbdb.LogicalSwitch{Name: switchName})
+			if err != nil {
+				return nil, fmt.Errorf("cannot fetch switch for node %s: %v", switchName, err)
+			}
+			ops, err = libovsdbops.CreateOrUpdateLogicalSwitchPortsOnSwitchOps(bsnc.nbClient, nil, logicalSwitch, lsps...)
```

Comment:
Right now we are overwriting the existing ops. 
While in this case it won't override previous entities (`addLogicalPortToNetwork()` does not create other entities) - I agree it's very bad practice..
Unfortunately passing the previous ops fails the transact:
```
UUID:{GoUUID:} Rows:[]} {Count:0 Error: Details: UUID:{GoUUID:} Rows:[]} {Count:0 Error: Details: UUID:{GoUUID:} Rows:[]}] and errors [named UUID "u0859861921" maps to UUID "31f7e730-aa11-4f5d-a422-afd29119c198" but found existing UUID "1f819417-21b6-4540-872b-4153f1d2259d"]: 1 ovsdb operations failed
```

The Solution I can offer is to move this function back to `addLogicalPortToNetwork` so that the ops are created only once. wdyt?